src/os/Transaction.cc

```diff
@@ -493,11 +493,13 @@ void Transaction::dump(ceph::Formatter *f)
         ghobject_t oid = i.get_oid(op->oid);
         uint64_t expected_object_size = op->expected_object_size;
         uint64_t expected_write_size = op->expected_write_size;
+        uint32_t alloc_hint_flags = op->alloc_hint_flags;
         f->dump_string("op_name", "op_setallochint");
         f->dump_stream("collection") << cid;
         f->dump_stream("oid") << oid;
         f->dump_stream("expected_object_size") << expected_object_size;
         f->dump_stream("expected_write_size") << expected_write_size;
+        f->dump_stream("alloc_hint_flags") << ceph_osd_alloc_hint_flag_string(alloc_hint_flags);
```

Comment:
great! can we use `dump_string()`? it is more light-weighted than `dump_stream()`.

Reply:
Thanks for pointed it out, I did not realized that.